README.md

```diff
@@ -9,20 +9,28 @@ and store as JSON.
 ## Run process
 
 In order for the process to access the syndication feed an API key is required.
-Details of registration are available on [NHS Choices](http://www.nhs.uk/aboutNHSChoices/professionals/syndication/Pages/Webservices.aspx).
-The application needs the api key available within the environment as the
-variable `SYNDICATION_API_KEY`.
+Details of registration are available on
+[NHS Choices](http://www.nhs.uk/aboutNHSChoices/professionals/syndication/Pages/Webservices.aspx).
+The application needs the api key available within the environment as the variable `SYNDICATION_API_KEY`.
 
-Running `npm start` will initiate the scrape. A successful scrape will result in the file `gp-data.json` being written to the `output` folder.
+Running `scripts/start` will bring up a docker container hosting a web server and initiate the scrape at a scheduled time.
+The default is 11pm. To test locally set an environment variable `ETL_SCHEDULE` to a new time,
+i.e. `export ETL_SCHEDULE='25 15 * * *'` to start the processing a 3:25pm.
+Further details available [here](https://www.npmjs.com/package/node-schedule)
 
-The ETL is now re-entrant - if the process is interrupted via `ctrl + c` while the ID list is being built, it will skip the pages or records it has already scanned. State is also persisted every 100 records in case of system failure.
+A successful scrape will result in the file `gp-data.json` being written to the `output` folder. This file may be downloaded from the web-server available at `http://localhost/json/gp-data.json`. Summary json is available at `http://localhost/json/summary.json`
 
-When the ETL has completed state information will remain in the output directory. Restarting will revisit any failed records that may have errored due to availabilty issues.
-To clear the state before starting run `npm run start-clear`.
+The ETL may also be run locally with `yarn start`
 
-To run the ETL end to end, but with only 3 pages of 90 surgeries, rather than the full 320+ pages run `npm run start-small`. Currenty these contain one record that returns
-a `500` error, so the retry behaviour can be observed.
-The small ETL can be run with the command `npm run start-small-clear` to remove any in progress files.
+The ETL is re-entrant - if the process is interrupted via `ctrl + c` while the ID list is being built, it will skip the pages or records it has already scanned. State is also persisted every 100 records in case of system failure.
+
+To clear the state before starting when running locally run `yarn run start-clear`. 
```

Comment:
Having the `*-clear` scripts is useful when developing and running locally. I'd suggest they are kept.